packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1842,6 +1842,10 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
     }
 
     const options = this.prepSectionDocOptions(declaration)
+    const isUploadButtonVisible = documentsSection.groups[0].fields.filter(
+      (field) => this.isVisibleField(field, documentsSection)
+    )
```

Comment:
```suggestion
    const isUploadButtonVisible = documentsSection.groups[0].fields.filter(
      (field) => this.isVisibleField(field, documentsSection)
    ).length > 0
```
The name implies the value to be a boolean so we should follow that